### PR TITLE
Adds r-bounds computations.

### PR DESCRIPTION
**Pull request by francocipollone**
# 🎉 New feature

Related to #15 

## Summary
Implements computation for the RBounds of the lane.

## Checklist
- [x] Signed all commits for DCO
- [x] Added tests
- [ ] Added example and/or tutorial
- [ ] Updated documentation (as needed)
- [ ] Updated migration guide (as needed)
- [ ] Consider updating Python bindings (if it affects the public API)

### CODE REVIEW

File: src/geometry/lane_geometry.cc [hunk 1]
```diff
@@ -117,5 +117,27 @@ maliput::math::Vector3 LaneGeometry::WInverse(const maliput::math::Vector3& xyz)
...
+  const maliput::math::Vector3 on_left = utility::InterpolatedPointAtP(left_, p_left);
+  const maliput::math::Vector3 on_right = utility::InterpolatedPointAtP(right_, p_right);
+  return {-(on_right - on_centerline).norm(), (on_left - on_centerline).norm()};
+}
+
+double LaneGeometry::FromCenterPToLateralP(const LineStringType& line_string_type, double p) const {
+  MALIPUT_THROW_UNLESS(line_string_type != LineStringType::kCenterLine);
+  const double lateral_length = line_string_type == LineStringType::kLeftBoundary ? left_.length() : right_.length();
+  const double p_equivalent = p * lateral_length / centerline_.length();
+  return std::clamp(p_equivalent, 0., lateral_length);
```
Comment (agalbachicar) [inline src/geometry/lane_geometry.cc:139]: missing algorithm maybe?

**Review by agalbachicar — DISMISSED** [review]
LGTM

I think I captured the discussion we had in https://github.com/maliput/maliput_sparse/issues/15#issuecomment-1250683237 . PTAL to it @francocipollone so we leave it (a docstring maybe?) in code.

File: src/geometry/lane_geometry.cc (hunk 1 above)
Comment (francocipollone) [reply src/geometry/lane_geometry.cc:144]: Done.

**Comment by francocipollone** [comment]
> PTAL to it @francocipollone so we leave it (a docstring maybe?) in code.

Done. I will add more info about orientation when superelevation computation is added.

**Review by agalbachicar — APPROVED** [review]
LGTM

**Pull request merged**